src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -157,7 +155,6 @@ using namespace DataSurfaces;
 // Use statements for access to subroutines in other modules
 using namespace ScheduleManager;
 using namespace SolarShading;
-using namespace DaylightingManager;
```

Comment:
Any `using namespace` going away makes me happy too.  Even though this is in the cc file and not a header, I bet @lefticus would still [agree](https://youtu.be/MZqjl9HEPZ8?t=566).